integreat_cms/static/src/css/tinymce_custom.css

```diff
@@ -10,3 +10,43 @@
     display: inline-block;
     white-space: pre-wrap;
 }
+
+div:has(> a[href^="https://integreat.app/"][href*="/contact/"]) [translate="no"],
+.contact-card [translate="no"] {
+    background-color: initial;
+    white-space: initial;
+}
+div:has(> a[href^="https://integreat.app/"][href*="/contact/"])[data-mce-selected]
+{
+    outline-width: 4px;
+}
+
+[contenteditable="false"] a {
+    pointer-events: none;
+}
+
+.contact-card {
```

Comment:
> Do you think it would make sense to move this to `style.scss` so that it applies to the other places that can render a contact card in the cms too (For example pdf export, page preview, and version history view don't render contacts correctly right now)?

Sure!

Though I think PDF will be its own topic (and likely also shouldn't use the same styling, I feel)

> Another idea would be to put it inline, since then the app would not have to have the same styles duplicated and the contact cards and the style could not get out of sync. But not sure how much effort that would be

I could not get it to work with tailwind classes last time (will try again though). Inline styles are an option, although discouraged

Reply:
OK, so after looking into this for a bit:

- the TinyMCE stylesheet does not include the large `style.scss`, and I am a bit hesitant to add it
- styling through Tailwind classes has no effect on the contact cards in the editor
- similar story with PDF

Looks like styling through inline styles, as Peter originally did, is the simplest way to solve this - except I do not know how that will impact PDF exports, since the xhtml2pdf library only supports a very limited subset of CSS3.

Not really sure how to proceed with this, do you have any advice?

Reply:
Then it would look like this in pdf: 
![image](https://github.com/user-attachments/assets/d1a769dc-f889-4fd0-a2cb-b0260b5dfe98)

Reply:
> Maybe we could only add inline style to the marker link to make it `display: none`

If you talk about adding `display: none` to the marker via the `style` attribute in-editor, e.g. both when rendering in tinymce and server side *(since the same will happen next time it is loaded into tinymce)*, my experience is that the browser recognizes the link as superflous and eliminates it from the DOM, so it also doesn't get included when then submitting the page translation form. This means the element is no longer recognized as a contact card by tinymce or the backend, and it also is no longer tracked by `linkcheck`. That's why I made the link merely very tiny and `opacity: 0`

Reply:
@PeterNerlich I remember we talked about this at the conference, but I could not replicate the issue. Granted, I have not tried with an inline style yet, but applying `display: none` via the stylesheet has no negative consequences.

Reply:
> Then it would look like this in pdf: ![image](https://private-user-images.githubusercontent.com/78504586/395151134-d1a769dc-f889-4fd0-a2cb-b0260b5dfe98.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMTYzMjcsIm5iZiI6MTczNDAxNjAyNywicGF0aCI6Ii83ODUwNDU4Ni8zOTUxNTExMzQtZDFhNzY5ZGMtZjg4OS00ZmQwLWEyY2ItYjAyNjBiNWRmZTk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEyVDE1MDcwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzNTE2YTQ1YzA3ZTE2ZjU1OTk3MDljNDE0MTM0ZGYxMmJkNzA2ZDU4ZTY0MTUyNjA2ZWVjOTNkYTJhOTFhMTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.vgSJoIw0N0LRXQN_AFXR0vpoSgaVof9-8fcT3raK2tg)

IMO that actually fits really well with the PDF. I'll try adding the inline style, hopefully it won't cause the mentioned issue, and implement your recommendation re: stylesheet imported by both relevant parent stylesheets